srcpkgs/next/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'next'
+pkgname=next
+version=1.3.2
+revision=1
+wrksrc="${pkgname}-${version}"
+build_style=gnu-makefile
+hostmakedepends="sbcl curl pkg-config"
+makedepends="webkit2gtk-devel libfixposix-devel libressl-devel"
+depends="sqlite xclip libfixposix-devel libressl-devel"
+short_desc="Keyboard-oriented, extensible web-browser"
+maintainer="0x0f0f0f <sudo-woodo3@protonmail.com>"
+license="BSD-3-Clause"
+homepage="https://next.atlas.engineer/"
+distfiles="https://github.com/atlas-engineer/next/archive/${version}.tar.gz"
+checksum=1c423f4af3b732f2b5bea74a571b17f51d416a7161cc0ad25574725612588593
+nostrip=true
+nopie=true
+nocross="https://travis-ci.org/void-linux/void-packages/builds/590346382"
```

Comment:
Looks easy to fix if we could patch, but we can't apparently. https://github.com/sionescu/iolib/blob/81e20614c0d27f9605bf9766214e236fd31b99b4/src/grovel/grovel.lisp#L298